TWLight/static/css/local-rtl.css

```diff
@@ -331,6 +331,14 @@ div.panel-body form.form-inline input#submit-id-submit.btn.btn-default {
   padding: 6px 25px 0px 0px;
 }
 
+.resource-label-danger {
+  background-image: url("../img/bullet-danger.png");
+  background-repeat: no-repeat;
+  background-position: top left; /* Must set explicitly for CSS Janus. */
```

Comment:
Looks like this should have a background-position of top right, since this is RTL (see L329)